### DISCUSSION TITLE
Adding Google Tasks

### DISCUSSION THREAD

**Discussion by selwynorren** [elementary/tasks#235] [General]
Hi All, I am pretty interested in this new tasks app as I used to use Planner quite a bit.

How would I like my Google Tasks with the Tasks App? I cant seem to find a way, online accounts does not provide a google option?

**Comment by marbetschar** [comment]
@selwynorren since the Online Acounts is pretty much non-functional as of now, you need to install [Evolution](https://wiki.gnome.org/Apps/Evolution/) and configure your Google Account there: `Edit > Accounts > Add > Collection Account`, enter your Google Username / E-Mail address and click `Look Up`. Click `Enter password` and follow the further instructions of the assistant.

Once setup is completed in Evolution, the Google Tasks should appear in elementary Tasks. From what I know pretty much everything should work with GTasks too. If you encounter any bugs or missing features, feel free to open an issue in this repository.

**Reply by selwynorren** [reply]
Thank you so much for this. Ill give that a try, pretty excited for this, especially the Google Tasks option

**Reply by selwynorren** [reply]
> @selwynorren since the Online Acounts is pretty much non-functional as of now, you need to install [Evolution](https://wiki.gnome.org/Apps/Evolution/) and configure your Google Account there: `Edit > Accounts > Add > Collection Account`, enter your Google Username / E-Mail address and click `Look Up`. Click `Enter password` and follow the further instructions of the assistant.
> 
> Once setup is completed in Evolution, the Google Tasks should appear in elementary Tasks. From what I know pretty much everything should work with GTasks too. If you encounter any bugs or missing features, feel free to open an issue in this repository.

100% works perfectly. Interestingly, I install evolution, added a Google Account, then loaded a google task using tasks, removed evolution, and it continues to work. So something that gets installed alongside evolution (Dependency) is what makes it work.

So thank so much for your help with this

**Reply by marbetschar** [reply]
You're welcome!

> So something that gets installed alongside evolution (Dependency) is what makes it work.

Yes, that's correct and expected: elementary uses the Evolution Data Server for syncing in Calendar, Mail and Tasks. Evolution itself is merely another UI for Evolution Data Server. Ultimately the goal is to provide account configuration in System Settings > Online Accounts of elementary itself so Evolution does not need to be installed at all.
